doc/R-vignette/igraph_extended_tutorial_R_vignette_english.Rmd

```diff
@@ -0,0 +1,682 @@
+
+| Keyword argument  | Purpose                                                                                                                                                                                                                                                           |
+|--------------------------------|----------------------------------------|
+| `autocurve.edges` | Whether to determine the curvature of the edges automatically in graphs with multiple edges.                                                                                                                                                                      |
+| `layout`          | The layout to be used. It can be an instance of `Layout`, a list of tuples containing X-Y coordinates, or the name of a layout algorithm. The default is `auto`, which selects a layout algorithm automatically based on the size and connectedness of the graph. |
+| `margin`          | The amount of empty space below, over, at the left and right of the plot, it is a numeric vector of length four.                                                                                                                                                  |
+
+### Specifying colors in plots
+
+igraph understands the following color specifications wherever it expects a color (e.g., edge, vertex or label colors in the respective attributes):
+
+***X11 color names***
+
+See the [list of X11 color names](https://en.wikipedia.org/wiki/X11_color_names) in Wikipedia for the complete list. Alternatively you can see the keys of the igraph.drawing.colors.known_colors dictionary. Color names are case insensitive in igraph so "DarkBlue" can be written as "darkblue" as well.
+
+***Color specification in CSS syntax***
+
+This is a string according to one of the following formats (where *R*, *G* and *B* denote the red, green and blue components, respectively):
+
+-   `#RRGGBB`, components range from 0 to 255 in hexadecimal format. Example: `"#0088ff"`.
+-   `#RGB`, components range from 0 to 15 in hexadecimal format. Example: `"#08f"`.
+-   `rgb(R, G, B)`, components range from 0 to 255 or from 0% to 100%. Example: `"rgb(0, 127, 255)"` or `"rgb(0%, 50%, 100%)"`.
+
+### Saving plots
+
+igraph can be used to create publication-quality plots. The preferred format is inferred from the extension. igraph can save to anything that is supported by Cairo, including SVG, PDF and PNG files. SVG or PDF files can then later be converted to PostScript (`.ps`) or Encapsulated PostScript (`.eps`) format if you prefer that, while PNG files can be converted to TIF (`.tif`):
+
+```{r echo=FALSE,eval=FALSE}
+png("social_network.png", 600, 600) 
+plot(g, layout =layout, vertex.size = 20, margin = 0.5,
+            vertex.color=c( "red", "yellow")[1+(V(g)$gender == "m")], 
+            edge.width=c(1,3)[1+(E(g)$is_formal == TRUE)]
+        )
```

Comment:
@adriandiazlab please add an option that offsets the vertex labels so they don't 100% overlap with the nodes. Thanks!